tests/python/pants_test/engine/legacy/test_graph_integration.py

```diff
@@ -23,41 +23,22 @@ def use_pantsd_env_var(cls):
     """
     return False
 
-  _WARN_FMT = """[WARN] Globs did not match. Excludes were: {excludes}. Unmatched globs were: {unmatched}."""
-
   _NO_BUILD_FILE_TARGET_BASE = 'testprojects/src/python/no_build_file'
 
   _SOURCES_TARGET_BASE = 'testprojects/src/python/sources'
 
-  _SOURCES_ERR_MSGS = {
-    'missing-globs': ("globs('*.a')", ['*.a']),
-    'missing-rglobs': ("rglobs('*.a')", ['**/*.a']),
-    'missing-zglobs': ("zglobs('**/*.a')", ['**/*.a']),
-    'missing-literal-files': (
-      "['another_nonexistent_file.txt', 'nonexistent_test_file.txt']", [
-      'another_nonexistent_file.txt',
-      'nonexistent_test_file.txt',
-    ]),
-  }
-
   _BUNDLE_TARGET_BASE = 'testprojects/src/java/org/pantsbuild/testproject/bundle'
-  _BUNDLE_ERR_MSGS = [
-    ['*.aaaa'],
-    ['**/*.aaaa', '**/*.bbbb'],
-    ['**/*.abab'],
-    ['file1.aaaa', 'file2.aaaa'],
-  ]
```

Comment:
Inlined. They weren't being used by multiple tests so inlining follows the principle of proximity.